ddl/db_partition_test.go

```diff
@@ -661,10 +661,11 @@ func (s *testIntegrationSuite5) TestAlterTableDropPartition(c *C) {
 	partition by range( hired ) (
 		partition p1 values less than (1991),
 		partition p2 values less than (1996),
-		partition p3 values less than (2001)
+		partition p3 values less than (2001),
+		partition p4 values less than (2006)
```

Comment:
beside this, you can write some dependent Test func about `drop multi partition and truncate multi partition`

Reply:
added in https://github.com/pingcap/tidb/pull/18593/commits/366ab3092be0d5a4f5c1b38661f0df8e280936e4